.github/workflows/spread-large.yaml

```diff
@@ -0,0 +1,50 @@
+name: Large Manual Spread
```

Comment:
pedantic: I think this name is misleading, since it's primarily a scheduled CI job, with the option to be triggered manually.

What about something like "scheduled spread tests"?

Reply:
Ah yes, this is a letfover from the charmcraft copy-and-paste. Will fix the entry and the file name as well.